apps/emqx_dashboard_sso/src/emqx_dashboard_sso_manager.erl

```diff
@@ -175,47 +184,60 @@ start_backend_services() ->
                     ?SLOG(error, #{
                         msg => "start_sso_backend_failed",
                         backend => Backend,
-                        reason => Reason
+                        reason => emqx_utils:redact(Reason)
```

Comment:
insert ets with a empty map `#{}` as state, and `last_error => emqx_utils:redact(Reason)` ?